### ISSUE TITLE
AV blocking release 2.3.0.1

### ISSUE THREAD

**Issue by Djinada** [rcmaehl/WhyNotWin11#221]
Following is blocked by F-Secure.

https://www.f-secure.com/v-descs/w32_malware.shtml

Edit: [variant] is W32/Malware!Deepguard.a!fsmind

**Comment by rcmaehl** [comment]
Duplicate of #49 . Please try the one without `x86` when possible

**Comment by Djinada** [comment]
This occurred with the 64-bit version, should've specified this.

**Comment by micwoj92** [comment]
I would still say that it is false positive. Contact the AV vendor and hope for quick fix.
